goon-universe/queries.js

```diff
@@ -70,7 +76,17 @@ function getTaskPage(scenarioID, callback){
             throw error
         }
         callback(results.rows)
-    }) 
+    })
+}
+
+function getConclusionPage(scenarioID, callback){
```

Comment:
Calling `GET /scenarios/conclusion` with `scenarioID: 1` returns a 404. Is the example data set up correctly to test this endpoint?

Reply:
Probably; I can double-check this once I update the example data

Reply:
Test data is not fully updated but this particular endpoint should work correctly now.

Reply:
@rasdy can you reverify when you have a chance?

Reply:
Works fine. Ping me on slack next time I'm slow to respond on github